src/ConfigAggregator.php

```diff
@@ -274,12 +277,18 @@ private function cacheConfig(array $config, $cachedConfigFile)
             return;
         }
 
-        $contents = sprintf(
-            self::CACHE_TEMPLATE,
-            get_class($this),
-            date('c'),
-            var_export($config, true)
-        );
+        $hasVarExporter = is_callable("Brick\VarExporter\VarExporter::export");
+
+        try {
+            $contents = sprintf(
+                self::CACHE_TEMPLATE,
+                get_class($this),
+                date('c'),
+                $hasVarExporter ? VarExporter::export($config) : var_export($config, true)
```

Comment:
I would argue we should pass the following flags to `VarExporter::export()`:

- `VarExporter::ADD_RETURN` would allow us to simplify the template
- `VarExporter::CLOSURE_SNAPSHOT_USES` would allow usage of `use` with closures and ensure arrow functions that are using variables from the current context work without issue. (I know I've actually done this a few times!)

Reply:
Good suggestions, added both flags.